vts/earsigner/keyloader_aws.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package earsigner
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/secretsmanager"
+)
+
+// AwsKeyLoader is IKeyLoader implementation that loads the key from AWS Secrets Manager.
+type AwsKeyLoader struct {
+	context context.Context
+}
+
+// NewAwsKeyLoader creates a new NewAwsKeyLoader using the specified Context.
+func NewAwsKeyLoader(c context.Context) *AwsKeyLoader {
+	return &AwsKeyLoader{c}
+}
+
+// Load they key from the specified URL. The url must be in the following format:
```

Comment:
```suggestion
// Loads the key from the specified URL. The url must be in the following format:
```

Reply:
The godoc convention is to start with the method name.

Reply:
I' ve fixed the the "they" --> "the" typo, but I kept it as "Load", as per the convention.